_config/site.yml

```diff
@@ -127,9 +127,9 @@ profiles:
 
   drupal_production:
     <<: *profile
-    base_url: http://developer-drupal.web.prod.ext.phx2.redhat.com/
+    base_url: http://rhdp-drupal.redhat.com/
     final_base_url: https://developers.redhat.com/
-    drupal_base_url: http://developer-drupal.web.prod.ext.phx2.redhat.com/
+    drupal_base_url: http://rhdp-drupal.redhat.com/
```

Comment:
Not needed, but it's fine. Awestruct doesn't talk to Drupal anymore.

Reply:
@LightGuard Yep thanks.

I've ensured the prod config on the VM has been updated with the new hostname, so I think we're good to go.

cheers,

Rob